dependabot/resources/extensions.json

```diff
@@ -740,6 +744,22 @@
                 "ballerina-distribution"
             ]
         },
+        {
```

Comment:
Add this to module_list.json file. This is auto generated file.

Reply:
Addressed the suggestions